runs/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
This should not be needed because the runs folder should be included in the base gitignore.

Reply:
actually it isn't
![image](https://github.com/user-attachments/assets/6effaa07-6dff-4f2c-ba7c-9bb15e8d1dae)

would you prefer to keep it in root `.gitignore`? I could do that